container/mesos/client_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mesos
+
+import "fmt"
+
+type mesosAgentClientMock struct {
```

Comment:
nit: We use "mock" to refer to a testing struct for which function calls and return values are explicitly specified in the test.  See https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cadvisor/testing/cadvisor_mock.go#L23, as an example.  Most use the github.com/stretchr/testify/mock library.

For testing structs like this one, which is a "fake" version of the real client, we refer to them as `fake`.

Reply:
I referred to the one in "https://github.com/google/cadvisor/blob/master/container/containerd/client_test.go#L24". Isnt it the preferred way ?

Reply:
Addressed it